extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -47,14 +53,25 @@ export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, W
 	}
 
 	protected async onOk(): Promise<void> {
-		await this.scriptToNotebook();
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
```

Comment:
Why would this check be necessary? This is the `DeployAzureSQLDBWizard` class - I would expect that the provider would always be of this type

Reply:
So, this deployment has additional webpage types where a page from azure portal is opened. Therefore, this check is necessary.